### PR TITLE
[16.0][ADD] stock_package_type_volume

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
This module adds the volume to the package type.

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Review by JordiMForgeFlow — APPROVED** [review]
code review

**Comment by rousseldenis** [comment]
@jbaudoux

**Review by AaronHForgeFlow — APPROVED** [review]
Code review LGTM

**Review by jbaudoux — APPROVED** [review]

File: stock_package_type_volume/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 ForgeFlow
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+{
+    "name": "Stock Package Type Volume",
+    "version": "16.0.1.0.0",
+    "summary": "Compute volume of a package type",
+    "category": "Warehouse",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "license": "LGPL-3",
```
Comment (jbaudoux) [inline stock_package_type_volume/__manifest__.py:10]: A reason for LGPL instead of AGPL ?

**Review by MiquelRForgeFlow — APPROVED** [review]

File: stock_package_type_volume/__manifest__.py (hunk 1 above)
Comment (JordiBForgeFlow) [reply stock_package_type_volume/__manifest__.py:10]: There are some custom modules depending on this one. That is the reason.

**Pull request merged**